src/subcommand/wallet/inscribe.rs

```diff
@@ -635,4 +639,30 @@ mod tests {
       error
     );
   }
+
+  #[test]
+  fn inscribe_with_specific_destination() {
```

Comment:
I think this test might actually be testing something that's already being tested, e.g. the specific destination address. For this, because it introduces a flag, the only test I think we need is an integration test which calls inscribe with `--destination`, and makes sure that the mined reveal transaction has the right destination.

Reply:
ack